layouts/partials/openapi/render-object-table.html

```diff
@@ -53,6 +53,11 @@
 
     {{ end }}
 
+    {{/* Enforce `additionalProperties: {}` instead of the equivalent `additionalProperties: true` for easier processing */}}
+    {{ if eq .additionalProperties true }}
+      {{ errorf "Use `additionalProperties: {}` instead of `additionalProperties: true`" }}
+    {{ end }}
```

Comment:
To cover more use cases, we can check whether the type of `additionalProperties` is a map with [`reflect.isMap`](https://gohugo.io/functions/reflect/ismap/).

Reply:
You mean throw the error if `additionalProperties` is not a map, regardless of what it is?